src/main/kotlin/frc/robot/subsystems/hood/HoodIO.kt

```diff
@@ -13,6 +13,8 @@ interface HoodIO {
 
     fun setAngle(angle: Measure<Angle>, torqueCompensation: Supplier<Double>) {}
 
+    fun setPID(kP: Double, kI: Double, kD: Double, kS: Double, kV: Double, kA: Double, kG: Double) {}
```

Comment:
I'd call all these functions `setGains` because some gains here aren't relevant to PID.

Reply:
6a4a0a99b9a4439040a4d528da90b0b8e1cb2b73